### ISSUE TITLE
documentation fixes: ofFileUtils.h

### ISSUE THREAD

**Issue by bakercp** [openframeworks/openFrameworks#4951]
Hey all, a few of these low level doc PRs got merged before I got to take a look at them -- for ofFileUtils.h a couple things -- 

To conform to doxygen style guide:
- Mixed use of \return \returns - both are valid, but style guide suggests \returns
- Need complete sentences with capitalization / punctuation, it's -> its where appropriate.
- 80 char hard breaks unless impossible.
- Need \brief line with a blank line after (for parsers -- \brief tag will be removed later) and order should 
be:

- brief
- blank
- description
- notes, warnings, throws, etc
- params
- returns

I'm happy to take a pass unless someone else wants to.

Thanks @danomatika for all of your work on these!

**Comment by bakercp** [comment]
@danomatika Do you want to take this on?  Otherwise, I'm happy to do it.

**Comment by danomatika** [comment]
What about \brief after \class? Whitespace there too?

**Comment by bakercp** [comment]
Yes, whitespace there too.

**Comment by danomatika** [comment]
So much for "minimal, 1 or 2 lines" :)

**Comment by bakercp** [comment]
Yes :), with that said, after a discussion w/ @workergnome and @HalfdanJ earlier today, we're planning to get rid of the \class tag, but I'd say for now leave it in and we'll remove it all at once when we run through a conformance check (@HalfdanJ is researching the possibility of a documentation linter).  Fun times are ahead ...

**Comment by danomatika** [comment]
I hope y'all aren't trying to outsmart doxygen too much ... sounds like heading down the road of "let's write out own documentation parser because the existing one isn't super pretty". :)

**Comment by danomatika** [comment]
https://github.com/danomatika/openFrameworks/commit/64301c30072605d6fe5070fec5bb739fb61c7658

**Comment by danomatika** [comment]
Closed with #4979
